app/src/main/java/org/oppia/app/profile/ProfileChooserViewModel.kt

```diff
@@ -36,6 +38,12 @@ class ProfileChooserViewModel @Inject constructor(
       ProfileChooserUiModel.newBuilder().setProfile(it).build()
     }.toMutableList()
 
+    profileList.forEach {
+      if (it.profile.avatar.avatarTypeCase.number == ProfileAvatar.AVATAR_COLOR_HEX_FIELD_NUMBER) {
```

Comment:
Should this also get updated as per this comment: https://github.com/oppia/oppia-android/pull/539/files#r356367436